lib/lhm/chunker.rb

```diff
@@ -16,7 +16,9 @@ class Chunker
     def initialize(migration, connection = nil, options = {})
       @migration = migration
       @connection = connection
-      @throttler = options[:throttler]
+      if @throttler = options[:throttler]
+        @throttler.connection = @connection if @throttler.respond_to?(:connection=)
```

Comment:
right.. perhaps the `run` command should receive the connection.
that would need a big refactor.